### PR TITLE
add repair guide for pod install failing.

### LINKED ISSUE #170: [Docs] pod install failing after adding livekit_client dependency
After adding the latest `livekit_client: ^1.1.2` on pubspec, I am getting this error on Mac M1:

```bash
cd ios
arch -x86_64 pod install

Analyzing dependencies
firebase_core: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
firebase_messaging: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
[!] CocoaPods could not find compatible versions for pod "livekit_client":
  In Podfile:
    livekit_client (from `.symlinks/plugins/livekit_client/ios`)

Specs satisfying the `livekit_client (from `.symlinks/plugins/livekit_client/ios`)` dependency were found, but they required a higher minimum deployment target.
```

flutter doctor:
```bash
[✓] Flutter (Channel stable, 3.0.5, on macOS 12.5.1 21G83 darwin-arm, locale en-BR)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.71.0)
[✓] Connected device (3 available)
[✓] HTTP Host Availability

• No issues found!
```

Firebase dependencies:
```yaml
  firebase_core: ^1.22.0
  firebase_messaging: ^11.4.4
```

Is there some missing steps in the docs?

### PR DESCRIPTION
**Pull request by cloudwebrtc**
close #170

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,23 @@ post_install do |installer|
 end
 ```
 
+When you use M1 Macs, you may get pod install failling when adding or upgrading `livekit_client`, you can use the steps below to fix it.
```
Comment (davidzhao) [inline README.md:90]:
```suggestion
For iOS, the minimum supported deployment target is `12.1`. You will need to add the following to your Podfile.
```

File: README.md [hunk 2]
```diff
@@ -87,6 +87,23 @@ post_install do |installer|
...
+
+1. Set the ios version on `Podfile` to `12.1`
+
+```xml
```
Comment (davidzhao) [inline README.md:94]:
```suggestion
```
```

File: README.md [hunk 3]
```diff
@@ -87,6 +87,23 @@ post_install do |installer|
 end
 ```
 
+When you use M1 Macs, you may get pod install failling when adding or upgrading `livekit_client`, you can use the steps below to fix it.
+
+1. Set the ios version on `Podfile` to `12.1`
```
Comment (davidzhao) [inline README.md:92]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -87,6 +87,23 @@ post_install do |installer|
...
+
+```xml
+platform :ios, '12.1'
+```
+
+2. Delete `Podfile.lock`
```
Comment (davidzhao) [inline README.md:98]:
```suggestion

You may need to delete `Podfile.lock` and re-run `pod install` after updating deployment target.
```

File: README.md [hunk 5]
```diff
@@ -87,6 +87,23 @@ post_install do |installer|
...
+
+3. Run pod install again
+
+```bash
+cd ios
+arch -x86_64 pod install --repo-update
+```
+
```
Comment (davidzhao) [inline README.md:106]:
```suggestion
```

Comment (cloudwebrtc) [reply README.md:105]: **Running pod install --repo-update in m1 mac will report an error**

image

**need to add prefix `arch -x86_64`**

image

Comment (cloudwebrtc) [reply README.md:105]: The latest `cocoapods` no longer require the command prefix `arch -x86_64` on m1, so your suggestion is correct.
https://stackoverflow.com/questions/64901180/how-to-run-cocoapods-on-apple-silicon-m1

**Pull request merged**